docs/UserGuide.md

```diff
@@ -123,6 +123,9 @@ All planned enhancements will also be listed in the [Planned Enhancements / Know
    - [Notifications](#notifications)
      - [Birthday notifications](#birthday-notifications)
    - [Payments](#payments)
+     - [Money Amount Format](#money-amount-format)
+      - [Pay a contact money: `pay`](#pay-a-contact-money-pay)
```

Comment:
skill issue

Reply:
Thanks! Fixed in 9a19b1bc394d3c2da11ab6a108eed7a6f80def25